source/dofs/dof_handler.cc

```diff
@@ -1649,6 +1653,17 @@ namespace internal
                             numbers::invalid_dof_index);
                       }
                 }
+
+            dof_handler.object_dof_ptr[l][d].push_back(
+              dof_handler.object_dof_indices[l][d].size());
```

Comment:
@Rombur These lines have been add in addition to the asserts. It was not noticed that the last entry of the CRS pointer was missing since currently only the pointer to the beginning of the *rows* is used.